www/src/views/starter-library/filtered-starters.js

```diff
@@ -241,7 +244,13 @@ export default class FilteredStarterLibrary extends Component {
               tag="button"
               overrideCSS={styles.loadMoreButton}
               onClick={() => {
-                this.setState({ sitesToShow: this.state.sitesToShow + 15 })
+                let showAll =
```

Comment:
Since there's a little more logic here now, maybe it makes sense to refactor to a class method? e.g.